libr/anal/sign.c

```diff
@@ -480,6 +480,40 @@ static void serialize(RAnal *a, RSignItem *it, char *k, char *v) {
 	}
 }
 
+static RList * deserialize_sign_space(RAnal *a, RSpace *space){
+	char k[R_SIGN_KEY_MAXSZ];
+	r_return_val_if_fail (a && space, NULL);
+
+	serializeKey (a, space, "", k);
+	SdbList *zigns = sdb_foreach_match (a->sdb_zigns, k, false);
+
+	SdbListIter *iter;
+	SdbKv *kv;
+	RList *ret = r_list_new ();
```

Comment:
This should probably be initialized with:
```suggestion
	RList *ret = r_list_newf ((RListFree)r_sign_item_free);
```
otherwise all the `RSignItem`s won't be freed, I think.